x/profile/keeper/msg_server_update_coordinator_description.go

```diff
@@ -0,0 +1,39 @@
+package keeper
+
+import (
+	"context"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/tendermint/spn/x/profile/types"
+)
+
+func (k msgServer) UpdateCoordinatorDescription(
+	goCtx context.Context,
+	msg *types.MsgUpdateCoordinatorDescription,
+) (*types.MsgUpdateCoordinatorDescriptionResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+
+	// Check if the coordinator address is already in the store
+	coordByAddress, found := k.GetCoordinatorByAddress(ctx, msg.Address)
+	if !found {
+		return &types.MsgUpdateCoordinatorDescriptionResponse{},
+			sdkerrors.Wrap(types.ErrCoordAddressNotFound, msg.Address)
+	}
+
+	coord := k.GetCoordinator(ctx, coordByAddress.CoordinatorId)
```

Comment:
If we can't find the coordinator while the address mapping exist in the store, this is an inconsistency issue

Inconsistency issue, that theorically cannot occur from the design of the app, must be handled with a `panic`

Reply:
```suggestion
	coord, found := k.GetCoordinator(ctx, coordByAddress.CoordinatorId)
```

I didn't notice.
The `Get<type>` method returns a boolean if the value is found or not. We must check this value to determine if the coordinator exist